fiware_cloto/cloto/tests/tests.py

```diff
@@ -104,14 +104,29 @@ def setUp(self):
         self.general = GeneralView()
 
     @patch('fiware_cloto.cloto.manager.InfoManager.logger')
-    def test_update_window(self, mock_logging):
-        # Create an instance of a GET request.
+    @patch('fiware_cloto.cloto.manager.InfoManager.pika')
+    def test_update_window(self, mock_pika, mock_logging):
```

Comment:
I miss a test in which you see that the window size really change for a specific tenantId, when you send the message.